src/components/core/styles/mixins/buttons.scss

```diff
@@ -113,28 +103,11 @@
     @include icon-button-variables;
 
     #{$button-selector} {
-      @include typo.text-xs--bold;
-      @include typo.font-smoothing;
-
-      text-decoration: none;
-      width: 100%;
       position: relative;
-      display: flex;
-      gap: var(--sbb-button-gap);
-      align-items: center;
-      justify-content: center;
-      text-align: left;
       transition-duration: var(--sbb-button-transition-duration);
       transition-timing-function: var(--sbb-button-transition-easing-function);
       transition-property: color;
-      min-height: var(--sbb-button-min-height);
       border-radius: var(--sbb-button-border-radius);
-
-      // The padding block value is only a minimal padding to preserve a padding if the content becomes larger than intended.
-      // Positioning of the content is made by flexbox vertical centering.
-      // The real padding displayed is larger than the defined value below.
-      padding-block: var(--sbb-button-padding-block-min);
-      padding-inline: var(--sbb-button-padding-inline);
```

Comment:
Are you sure they are not necessary anymore? Just remembering somehow they were needed.

Reply:
I remember I saw no diffs on chromatic, but i could recheck

Reply:
I need to test manually. Not sure if it was in chromatic. I remeber that I once added this. Will update you.

Reply:
It's ok, is only needed for the real button :)